policy-controller/k8s/api/src/labels.rs

```diff
@@ -38,17 +39,10 @@ pub struct Selector {
 // === Selector ===
 
 impl Selector {
-    pub fn from_expressions(exprs: Expressions) -> Self {
+    pub fn new(labels: Option<Map>, exprs: Option<Expressions>) -> Self {
         Self {
-            match_labels: None,
-            match_expressions: Some(exprs),
-        }
-    }
-
-    pub fn from_map(map: Map) -> Self {
-        Self {
-            match_labels: Some(map),
-            match_expressions: None,
+            match_labels: labels,
+            match_expressions: exprs,
         }
```

Comment:
While this is basically fine, I don't think there's any need to have to specify the options in all of the constructors. Small suggestion in https://github.com/linkerd/linkerd2/pull/7942

Reply:
Thanks for the suggestions, will merge that in :+1: